### DISCUSSION TITLE
getting started example confusion regarding github hook ingress

### DISCUSSION THREAD

**Discussion by tuananh** [tektoncd/triggers#1236] [Q&A]
i don't understand the ingress here https://github.com/tektoncd/triggers/blob/main/docs/getting-started/ingress-run.yaml#L19 in the getting started example

from what i understand, it will be the endpoint for github triggers but i dont quite get where teh service name values `getting-started` and port `8080` coming from

**Comment by dibyom** [comment]
Hello, 
When you create an EventListener, the Triggers reconciler will create a deployment and a service (the EL "sink") -- the service in the ingress-run corresponds to this EL service (which as you pointed out is the endpoint for the triggers). 

See the actual ingress task [here](https://github.com/tektoncd/triggers/blob/main/docs/getting-started/create-ingress.yaml) and docs for EL deployment/service [here](https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md#understanding-the-deployment-of-an-eventlistener)
